Manual.md

```diff
@@ -787,28 +788,73 @@ should be listed in `checkdepends` and will be installed as if they were part of
   a D-Bus session for applications that need it
 - `git`: some test suites run the `git` command
 
+<a id="deps_runtime"></a>
 Lastly, a package may require certain dependencies at runtime, without which it
 is unusable. These dependencies, when they aren't detected automatically by
-XBPS, should be listed in `depends`. This is mostly relevant for Perl and Python
-modules and other programs that use `dlopen(3)` instead of dynamically linking.
+XBPS, should be listed in `depends`.
+
+Libraries linked by ELF objects are detected automatically by `xbps-src`, hence they
+must not be specified in templates via `$depends`. Variable should list:
+
+- executables called as separate processes.
+- ELF objects using dlopen(3).
+- non ELF objects, i.e perl/python/ruby/etc modules.
```

Comment:
```suggestion
- non-ELF objects, i.e perl/python/ruby/etc modules.
```

Reply:
Thanks for remarks.

Here also abstracted over object format.